main.go

```diff
@@ -37,33 +38,77 @@ type Server struct {
 // Start runs a MongoDB server at a given version using a random free port
 // and returns the Server.
 func Start(ctx context.Context, version string) (*Server, error) {
+	return StartWithOptions(ctx, version)
```

Comment:
yes, as you prob gathered I didn't add any since it just calls StartWithOptions which is what the tests address. But let me see what I can do just to remove the greyed-out aspect